presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConfig.java

```diff
@@ -64,6 +64,7 @@
     private int maxPartitionBatchSize = 100;
     private int maxInitialSplits = 200;
     private int splitLoaderConcurrency = 4;
+    private int maxSplitsPerSec = 4;
```

Comment:
Default should be no throttling (Integer.MAX_VALUE.. or 0).

We generally do not abbreviate names ("max" is kind of an exception) so this should be "maxSplitsPerSecond"

BTW I think we don't need to have 0 as a special value, so you would have `@Min(1)` on `getMaxSplitsPerSec`

Reply:
Thanks @findepi for your review. I meant to set 0 as the default value but made a mistake while rebasing.
I will set Integer.MAX_VALUE as default, this will however add the overhead even in the case where throttling is not desired, but it should be minor.